src/structures/MessageEmbed.js

```diff
@@ -198,11 +199,9 @@ class MessageEmbed {
    * @returns {MessageEmbed}
    */
   attachFiles(files) {
+    files = files.map(file => file instanceof MessageAttachment ? file.file : file);
     if (this.files) this.files = this.files.concat(files);
```

Comment:
As `this.files` is not longer `undefined` but always truthy, this (and the next line) can be simplified to just

```javascript
this.files = this.files.concat(files);
```

And additionally, you can also inline the previous line to avoid argument re-assignment.